source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -71,6 +71,10 @@ connection behavior:
      - Applies the ``ConnectionPoolSettings.Builder`` block and then sets the
        :ref:`connection pool settings <mcs-connectionpool-settings>`.
 
+   * - ``applyToLoggerSettings()``
+     - Applies the ``ConnectionPoolSettings.Builder`` block and then sets the
```

Comment:
Q: should this say `LoggerSettings.Builder`?

```suggestion
     - Applies the ``LoggerSettings.Builder`` block and then sets the
```

Reply:
Yes - copy paste error.